### ISSUE TITLE
eshopconatiners on WSL2 

### ISSUE THREAD

**Issue by sunnyggadekar** [dotnet/eShopOnContainers#1641]
Hi any documentation or prerequisite on running eShopOnContainers on WSL2 ? thanks in advance

**Comment by sughosneo** [comment]
Hi @sunnyggadekar , thank you for reaching out.

You can run the `eshopconatiners` with docker configured with WSL2. Make sure you have [installed](https://docs.docker.com/docker-for-windows/install/) and [configured](https://github.com/dotnet-architecture/eShopOnContainers/wiki/Windows-setup#configure-docker) docker in your environment. After that, you can run the below commands from the **/src/** directory and get started with the `eShopOnContainers` immediately.

```powershell
docker-compose build
docker-compose up
```

You should be able to browse different components of the application by using the below URLs :

```
Web Status : http://host.docker.internal:5107/
Web MVC :  http://host.docker.internal:5100/
Web SPA :  http://host.docker.internal:5104/
```

>Note: If you are running this application in macOS then use `docker.for.mac.localhost` as DNS name in `.env` file and the above URLs instead of `host.docker.internal`.

Hope this helps !

**Comment by sunnyggadekar** [comment]
Hey @sughosneo eShopOnContainers worked on WSL2 :) 

-installed WSL2
-followed the above mentioned steps
-encountered Login failed for user 'sa'. Reason: Failed to open the explicitly specified database 'Microsoft.eShopOnContainers.Service.IdentityDb'
-which was resolved by  #1590 
-all services are up and running

Thanks

**Comment by sughosneo** [comment]
Great! Thank you @sunnyggadekar,  for the update. 
 I am closing this issue as of now. Please feel free to reopen it, if needed.
